### ISSUE TITLE
Error: Could not find operation type

### ISSUE THREAD

**Issue by vibhanshu909** [AlecAivazis/houdini#98]
I'm using houdini with svelte-kit.

I have a graphql endpoint that only contains **queries**, no **mutations**, no **subscriptions**.
I don't get any error running `npx houdini generate -p`.
But when I run `npm run dev`, I'm getting the following error on the webpage:

```sh
Error: Could not find operation type
    at file:///home/anonymous/Documents/Sveltekit_Projects/e-commerce/node_modules/houdini-preprocess/build/esm/index.js:33667:31
    at step (file:///home/anonymous/Documents/Sveltekit_Projects/e-commerce/node_modules/houdini-preprocess/build/esm/index.js:109:23)
    at Object.next (file:///home/anonymous/Documents/Sveltekit_Projects/e-commerce/node_modules/houdini-preprocess/build/esm/index.js:90:53)
    at file:///home/anonymous/Documents/Sveltekit_Projects/e-commerce/node_modules/houdini-preprocess/build/esm/index.js:83:71
    at new Promise (<anonymous>)
    at __awaiter (file:///home/anonymous/Documents/Sveltekit_Projects/e-commerce/node_modules/houdini-preprocess/build/esm/index.js:79:12)
    at mutationProcessor (file:///home/anonymous/Documents/Sveltekit_Projects/e-commerce/node_modules/houdini-preprocess/build/esm/index.js:33656:12)
    at file:///home/anonymous/Documents/Sveltekit_Projects/e-commerce/node_modules/houdini-common/build/esm/index.js:114:42
    at step (file:///home/anonymous/Documents/Sveltekit_Projects/e-commerce/node_modules/houdini-common/build/esm/index.js:66:23)
    at Object.next (file:///home/anonymous/Documents/Sveltekit_Projects/e-commerce/node_modules/houdini-common/build/esm/index.js:47:53)
```

**Comment by AlecAivazis** [comment]
Ah damn, sorry you ran into that. I opened #99 which should fix this issue.

Thanks for reporting it!

**Comment by AlecAivazis** [comment]
This should now be fixed in `v0.7.1`

**Comment by vibhanshu909** [comment]
Great. Thanks!! :+1:
